src/util/piped_process.cpp

```diff
@@ -193,13 +206,7 @@ piped_processt::piped_processt(const std::vector<std::string> commandvec)
   start_info.hStdOutput = child_std_OUT_Wr;
   start_info.hStdInput = child_std_IN_Rd;
   start_info.dwFlags |= STARTF_USESTDHANDLES;
-  // Unpack the command into a single string for Windows API
-  std::wstring cmdline = widen(commandvec[0]);
-  for(int i = 1; i < commandvec.size(); i++)
-  {
-    cmdline.append(L" ");
-    cmdline.append(quote_windows_arg(widen(commandvec[i])));
-  }
+  const std::wstring cmdline = process_windows_args(commandvec);
```

Comment:
:pick: `process` is a little non-descriptive. Maybe `prepare_windows_command_line` would be better.